src/fiatconnect/saga.ts

```diff
@@ -50,6 +68,67 @@ export function* watchFetchFiatConnectQuotes() {
   yield takeLeading(fetchFiatConnectQuotes.type, handleFetchFiatConnectQuotes)
 }
 
+export function* handleStartFiatConnectTransfer({
+  payload: params,
+}: ReturnType<typeof startFiatConnectTransfer>) {
+  const { flow, quoteId, fiatConnectQuote, fiatAccountId } = params
+
+  if (flow === CICOFlow.CashOut) {
+    try {
+      Logger.info(TAG, 'Starting transfer out..')
+      const transferResult: TransferResponse = yield call(
+        doTransferOut,
+        fiatConnectQuote,
+        fiatAccountId
+      )
+
+      Logger.info(
+        TAG,
+        'Transfer out $succeeded. Starting transaction..',
+        JSON.stringify(transferResult)
+      )
+
+      const tokenList: TokenBalance[] = yield select(tokensListSelector)
+      const tokenInfo = tokenList.find((token) => token.symbol === fiatConnectQuote.getCryptoType())
+
+      if (!tokenInfo) {
+        throw new Error(`Invalid token: ${fiatConnectQuote.getCryptoType()}`)
+      }
+
+      const feeEstimates: FeeEstimatesState['estimates'] = yield select(feeEstimatesSelector)
+      const feeInfo = feeEstimates[tokenInfo.address]?.[FeeType.SEND]?.feeInfo
```

Comment:
Is it possible for this to be `undefined`? If so, should we guard against it and throw just in case, since we have a non-null assertion on it below?

Reply:
I don't think so, since withdraw amount page should do a fee estimation. But yeah we can do a similar check to tokenInfo